README.md

```diff
@@ -122,13 +122,29 @@ Compiling requires using CMake 3.1 or newer. GCC and Clang are both known to wor
 
 #### Docker building
 
-The recommended way to build for most platforms is to use Docker. Several Docker images are provided that contain the requisite toolchain and dependencies for building mGBA across several platforms.
+The recommended way to build for most platforms is to use Docker. Several Docker images are provided that contain the requisite toolchain and dependencies for building mGBA across several platforms. 
+
+Note: If you are on Windows, you need to make sure you are within the mGBA repository root directory, and it is located in a directory where it's in a *nix path. Docker is known to be stringent on this restriction. (See issue [#1985](https://mgba.io/i/1985) for details.) 
```

Comment:
This is still not technically accurate. The problem is that by default, Docker is restricted to being able to load things out of your home directory (C:\Users\<User>), but this can be overridden. You might need to search around for more info, and it might have been changed in the meantime. Did you run into this issue specifically?

Reply:
Alright, I might need to look into that more. That said, uh, the directory you listed isn't inside the home directory.

Reply:
Ahhh, when I reverted some changes, it reverted the `/usr` Will fix.

Reply:
I don't think you understand what I mean. It cares about the *Windows* home directory, not the "*NIX" home directory, which is a concept Docker for Windows has no idea about.

Reply:
Wait, let me do some experiments first.

Reply:
A bit of research shows:

- Docker Toolbox (or basically, Docker running on Windows systems older than Windows 10) has this limitation where you cannot run Docker images outside of the User home directory, unless you configure it or you put the image inside a VirtualBox container with a shared folder configured. Shared folders will map folder path A to folder path B. 

- Docker Desktop for Windows 10 and up have better native integration, and can run Docker images anywhere, so as long as you use `--mount` (Bind Mounts) or `--volume` (Volumes), the `source` and `target` folder destinations can be specified to be mapped as shared folders. This mounting is the same as mapping folder path A to folder path B.

https://forums.docker.com/t/mounting-folders-outside-the-users-folder-on-windows/65662/3

So it is wrong of me to assume that we need to put the mGBA checkout repository inside a "*nix" home directory, but rather, just specify the Docker command as mentioned in the README.md. It's only when you are using an older Windows system, do you then need to rely on VirtualBox shared folder configurations.

I have updated the README.md to reflect upon this new information.